src/pat/contentbrowser/README.md

```diff
@@ -81,3 +82,44 @@ Show a widget to select items in an offcanvas miller-column browser.
     data-pat-contentbrowser='{"selectableTypes": ["Image", "File"], "vocabularyUrl": "contentbrowser-test.json", "upload": true}'
 />
 ```
+
+## Register custom component
+
+Currently only for `SelectedItem` component available.
+
+```html
+<input
+    type="text"
+    class="pat-contentbrowser"
+    data-pat-contentbrowser='{
+        "customComponentKeys": {
+            "SelectedItem": "pat-contentbrowser.myfield.MySelectedItemComponent",
```

Comment:
Regarding the custom component keys  - isn't `selectedItem` the only component key which is supported yet?
I'm a bit restrained to use JSON structures as pattern options - especially JSON objects. Until now I had the impression that JSON objects prevent us from using the Patterns options parser and thus prevent us currently from using the BasePattern. But your PR here shows that this is not entirely true - You're using both the Patterns Parser and BasePattern here. However, at least the `customComponentKeys` is currently not formally defined in the Patterns Parser.
Probably all is good, but I'd have maybe made a flat single option out for the selectedItem component. Something like `selectedItemComponent`.